src/core/execute.h

```diff
@@ -360,6 +360,8 @@ struct ExecContext {
         Set *import_credentials;
 
         ImagePolicy *root_image_policy, *mount_image_policy, *extension_image_policy;
+
+        RuntimeScope runtime_scope;
```

Comment:
hmm, this does not belong here. It's not a user configurable facet. It's part of the execution context. Hence really should be part of ExecParameters.

Reply:
so i take it you moved it into the context because you want to know it for logging, to generate the right properties?

Maybe instead add a new structure for that: ManagerContext or so? And that carries the runtime scope, as well as the unit log field to use. Then embed that into the Manager object, and have another copy in the executors. And add a pointer from the Unit to it so that you have a uniform way to find it? or something like that?

Reply:
mmh that's a bit too many layers of indirection - yes I need it only for the unit logging, but I can simplify this a lot by just adding the right fields directly to the unit object, after all they are fixed per-unit anyway